frontend/src/metabase/questions/containers/CollectionEditorForm.jsx

```diff
@@ -41,12 +42,14 @@ export default class CollectionEditorForm extends Component {
                 form
                 title={fields.id.value != null ? fields.name.value : "New collection"}
                 footer={[
-                    <Button className="mr1" onClick={onClose}>
-                        Cancel
-                    </Button>,
-                    <Button primary disabled={invalid} onClick={handleSubmit}>
-                        { fields.id.value != null ? "Update" : "Create" }
-                    </Button>
+                    <div ref={(ref) => { this.footerRef = ref}}>
```

Comment:
Yeah it definitely feels silly. I think unfortunately some kind of reference to the Modal content inside this component is the only way to get this test to work with how our Modals are currently structured.